### PR TITLE
Update ReadMe

### PR DESCRIPTION
**Pull request by infotexture**
Clarify purpose & installation process.

### CODE REVIEW

File: .prettierrc.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ arrowParens: avoid
 bracketSpacing: false
 jsxBracketSameLine: false
 printWidth: 80
-proseWrap: never
+proseWrap: "preserve"
```
Comment (infotexture) [inline .prettierrc.yaml]: Based on recent experience in other projects, I think this is the better default setting.

File: package.json [hunk 2]
```diff
@@ -2,9 +2,9 @@
   "name": "prettier-setup",
   "license": "MIT",
   "devDependencies": {
+    "eslint": "4.15.0",
```
Comment (infotexture) [inline package.json]: Didn't mean to change this myself, looks like Prettier decided to rearrange things…

File: .prettierrc.yaml (hunk 1 above)
Comment (lipis) [reply .prettierrc.yaml]: Yes, why not.. no need for quotes in `yaml`

**Comment by lipis** [comment]
Thanks Roger! 🍺

**Pull request merged**